java-checks/src/main/java/org/sonar/java/checks/AssertionsInTestsCheck.java

```diff
@@ -79,40 +71,33 @@
     method("org.assertj.core.api.Fail", "shouldHaveThrown").withAnyParameters(),
     method("org.assertj.core.api.Assertions", STARTS_WITH_FAIL).withAnyParameters(),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "org.assertj.core.api.Assertions" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1192)